tests/tekton-resources/tasks/setup/eks/awscli-cp-with-pcp.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cluster-create-with-pcp-stack
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS cluster.
+    This Task can be used to create an EKS cluster for a given service role, region in an AWS account
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: kubernetes-version
+    default: "1.23"
+    description: The EKS version to install.
+  - name: region
+    default: "us-west-2"
+    description: The region where the cluster is in.
+  - name: endpoint
+    default: ""
+    description: "aws eks enpoint to create clusters against"
+  - name: service-role-name
+    description: servicerole name to be used for eks cluster to perform operations in customer account to setup cluster
+  - name: vpc-stack-name
+    description: The name of the VPC name you want to use for EKS cluster.
+  - name: aws-ebs-csi-driver-version
+    default: release-1.53
+    description: The release version for aws ebs csi driver.
+  - name: aws-pod-identity-agent-version
+    default: v1.3.9-eksbuild.1
```

Comment:
same with this . check if this upto date.

Reply:
I have updated to `release-1.53` for ebs volume already and have removed pod-identity addon as it is not used in this test